addons/customize-avatar-border/addon.json

```diff
@@ -88,7 +88,7 @@
       "id": "outline-color",
       "type": "color",
       "allowTransparency": true,
-      "default": "#4d97ff40",
+      "default": "#396db540",
```

Comment:
Why are some of the colors like this one still blue?

Reply:
🤦 That's what happens when I rush for no reason. I pasted the wrong colour everywhere.